spec/features/idv/in_person_spec.rb

```diff
@@ -176,6 +177,29 @@
     end
   end
 
+  context 'after in-person proofing is completed and passed for a partner' do
+    let(:sp) { nil }
+    before do
+      create_in_person_ial2_account_go_back_to_sp_and_sign_out(sp)
+    end
+
+    [
+      :oidc,
+      :saml,
+    ].each do |service_provider|
```

Comment:
Do you think there's value in exercising this for both SP protocols, or could we just do one? Reason I ask is that these tests can be quite slow to run.